extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -5,16 +5,45 @@
 class CommunityPageSpecialUsersModel {
 	const TOP_CONTRIB_MCACHE_KEY = 'community_page_top_contrib';
 	const ALL_ADMINS_MCACHE_KEY = 'community_page_all_admins';
+	const MODERATORS_MCACHE_KEY = 'community_page_moderators';
 	const GLOBAL_BOTS_MCACHE_KEY = 'community_page_global_bots';
 	const ALL_BOTS_MCACHE_KEY = 'community_page_all_bots';
 	const ALL_BLACKLISTED_IDS_MCACHE_KEY = 'community_page_all_blacklisted_ids';
 	const ALL_MEMBERS_MCACHE_KEY = 'community_page_all_members';
 	const ALL_MEMBERS_COUNT_MCACHE_KEY = 'community_page_all_members_count';
 	const RECENTLY_JOINED_MCACHE_KEY = 'community_page_recently_joined';
-	const MCACHE_VERSION = '1.2';
+	const MCACHE_VERSION = '1.3';
 
 	const ALL_CONTRIBUTORS_MODAL_LIMIT = 50;
 
+	// order of permissions is consistent with badges hierarchy in Discussions
+	const PERMISSION_HIERARCHY = [
+		'sysop',
+		'threadmoderator',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "threadmoderator" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)